css-size-adjust-1/Overview.bs

```diff
@@ -75,20 +82,24 @@ Values {#values}
   also accept the <a>CSS-wide keywords</a> keywords as their property value.
   For readability they have not been repeated explicitly.
 
-Default size adjustment {#default-adjustment}
-=============================================
+Size adjustment {#default-adjustment}
+=====================================
 
 This section defines the default size adjustment rules.  These rules
 are then referenced by the definition of the 'text-size-adjust' property
 in the following section.
 
-Issue: All of the subsections of this section need significant
-refinement:  additional detail, verification that the detail already
-present is correct, etc.
+The result of the adjustment must be a computed value of the 'font-size'
+property and/or the 'line-height' property.
+
+Note: Therefore, this adjustment is polyfill-able.
 
-Issue: It's not clear how much this section should define
-precise behavior versus how much it should allow future room for
-innovation and improvement.
```

Comment:
I think it might still be worth leaving this issue for now.  Your edits certainly change the balance on this question, but it might still be worth thinking about further.